client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,33 @@
-import React from 'react';
+import React from "react";
+import { Link } from "react-router-dom";
 
 const MovieCard = props => {
-  return;
+  const { title, director, metascore, stars } = props.movie;
+  return (
+    <div className="save-wrapper">
+      <Link to={`/movies/${props.movie.id}`}>
```

Comment:
```suggestion
      <Link to={`/movies/${id}`}>
```
id is not destructed from props.movie. minor improvement.